packages/@aws-cdk/aws-codedeploy/lib/trigger-configuration.ts

```diff
@@ -0,0 +1,20 @@
+import * as cdk  from '@aws-cdk/core';
+
+export interface TriggerConfiguration {
+  readonly triggerName: string;
+  readonly triggerTargetArn: cdk.Arn;
```

Comment:
1. I don't think `Arn` is the type you want here. `Arn` is just a collection of static utility methods with a private constructor, so it doesn't really make much sense here.
2. I wonder - what are the possible values that can be put here? ARNs of what resources can be legally passed here?

Reply:
1. I agree, I wasn't sure about this type either.
2. This property should be a reference to SNS topic. In CloudFormation templates we specify SNS topic Arn, maybe I should change the type to `aws-sns.Topic` and make it optional. If topic is not specified we can create one with default settings.

Reply:
2. In that case, you should use the `ITopic` interface from the `@aws-cdk/aws-sns` module.